src/test/java/uk/gov/hmcts/reform/sscs/service/CaseFlagMappingServiceTest.java

```diff
@@ -0,0 +1,73 @@
+package uk.gov.hmcts.reform.sscs.service;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringRunner;
+import uk.gov.hmcts.reform.sscs.ccd.domain.Appeal;
+import uk.gov.hmcts.reform.sscs.ccd.domain.HearingOptions;
+import uk.gov.hmcts.reform.sscs.ccd.domain.SscsCaseData;
+import uk.gov.hmcts.reform.sscs.ccd.domain.YesNo;
+import uk.gov.hmcts.reform.sscs.model.DescendantCaseData;
+import uk.gov.hmcts.reform.sscs.model.HearingWrapper;
+
+import java.util.List;
+import java.util.Map;
+
+@SpringBootTest
+@RunWith(SpringRunner.class)
+public class CaseFlagMappingServiceTest {
```

Comment:
Using an integration test for this is a bit overkill.. unit test should suffice.